libutils/json.h

```diff
@@ -544,6 +545,18 @@ typedef JsonElement *JsonLookup(void *ctx, const char **data);
   */
 JsonParseError JsonParse(const char **data, JsonElement **json_out);
 
+/**
+  @brief Parse the whole string to create a JsonElement
+  @param data [in] Pointer to the string to parse
+  @param json_out Resulting JSON object
+  @note In contrast to JsonParse(), this function will return
+        JSON_PARSE_ERROR_INVALID_END if there are trailing non-whitespace
+        characters after termination of the JsonElement in the remainder of
+        the string.
+  @returns See JsonParseError and JsonParseErrorToString
+  */
+JsonParseError JsonParseAll(const char **const data, JsonElement **const json_out);
```

Comment:
Same as in the recent blog post -- half of these `const` don't make sense in the declaration (prototype) of the function.

Reply:
Fixed ;)